crypto.c

```diff
@@ -115,8 +121,16 @@ smtp_init_crypto(int fd, int feature, struct smtp_features* features)
 
 	/* XXX clean up on error/close */
 	/* Init SSL library */
+#if (OPENSSL_VERSION_NUMBER >= 0x10100000L)
+	error = OPENSSL_init_ssl(0, NULL);
```

Comment:
I think we can just leave this out in fact,
>       As of version 1.1.0 OpenSSL will automatically allocate all resources
>       that it needs so no explicit initialisation is required. Similarly it
>       will also automatically deinitialise as required.

Reply:
We can today, but it might cause other issues later (there's talk of refactoring locking upstream to deal with performance issues). Oh well, I'll nuke it...

Reply:
I guess I was technically doing that already in my demo program. LOL.